ts/components/services/ContactPreferencesToggles/__test__/ContactPreferencesToggles.test.tsx

```diff
@@ -88,6 +89,66 @@ describe("ContactPreferencesToggles component", () => {
       // ).toBeDefined();
     });
   });
+
+  describe("when channels are loading", () => {
+    it("should render activity indicator on inbox", () => {
+      const initialState = appReducer(
+        undefined,
+        loadServicePreference.success({
+          id: "some_id" as ServiceId,
+          kind: "success",
+          value: { inbox: true, email: true, push: true, settings_version: 0 }
+        })
+      );
+      // the store will be in someLoading
+      const state = appReducer(
+        initialState,
+        loadServicePreference.request("aServiceID" as ServiceId)
+      );
+      const mockStore = configureMockStore<GlobalState>();
+      const store = mockStore({
+        ...state
+      } as GlobalState);
+      const component = renderComponent(store, {
+        channels: [
+          NotificationChannelEnum.EMAIL,
+          NotificationChannelEnum.WEBHOOK
+        ]
+      });
+      expect(
+        component.getByTestId("contact-preferences-inbox-switch-loading")
+      ).toBeDefined();
+      expect(
+        component.getByTestId("contact-preferences-webhook-switch-loading")
+      ).toBeDefined();
+    });
+  });
+
+  describe("when channels are loading", () => {
```

Comment:
this `describe` could be merged with the identical one [here](https://github.com/pagopa/io-app/pull/3229/files#diff-cb2a71e683a73f5ba90fb9b7133dcec5d71e416509c6c28319ad382c280ff83aR93)

Reply:
done in https://github.com/pagopa/io-app/pull/3229/commits/ff15844eb255bf3fefe0cbd937c0c7a0e923ca83